pkg/kubernetes/kubernetes.go

```diff
@@ -80,130 +84,167 @@ const (
 	backoffInterval   = 5 * time.Second
 	backoffMaxRetries = 5
 
-	requestTimeout  = 5 * time.Second
-	maxIdleConns    = 1
-	idleConnTimeout = 10 * time.Second
+	// requestTimeout  = 5 * time.Second
+	// maxIdleConns    = 1
+	// idleConnTimeout = 10 * time.Second
+
+	defaultQPSLimit   = 100
+	defaultBurstLimit = 150
```

Comment:
Also - just out of curiosity, how did we come up with these values?

Reply:
these ones were copied from `pkg/kubernetes/client/` layer and used in API requests to K8S. Have no clue how they were calculated